packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -350,6 +354,10 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
     }
 
     public createChannel(id: string = uuid(), type: string): IChannel {
+        if (id.includes("/")) {
+            throw new UsageError(`Channel id cannot contain slashes: ${id}`);
```

Comment:
nit: I think this should just say `Id cannot contain slashes` similar to the data store errors. The term channel doesn't mean anything to users.